gax-grpc/src/main/java/com/google/api/gax/grpc/OperationFuture.java

```diff
@@ -27,116 +27,28 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package com.google.api.gax.grpc;
+package com.google.api.gax.rpc;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import com.google.api.core.ApiFunction;
 import com.google.api.core.ApiFuture;
-import com.google.api.core.ApiFutures;
 import com.google.api.core.BetaApi;
 import com.google.api.gax.retrying.RetryingFuture;
-import com.google.longrunning.Operation;
-import com.google.protobuf.Message;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.Executor;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
 
-/**
- * An ApiFuture which tracks polling of a service, typically started by {@link
- * OperationCallable#futureCall(Object, CallContext)}. The polling is done periodically, based on
- * the {@link com.google.api.gax.retrying.TimedRetryAlgorithm} specified in {@link
- * OperationCallSettings} provided during creation of the corresponding {@link OperationCallable}.
- *
- * <p>This class is thread-safe.
- */
+/** An ApiFuture which provides access to additional operation data. */
```

Comment:
This was removed during a transition state when not all of the referenced types were present - in the current state, it would make sense to revert the documentation back, so I'll do that.